### PR TITLE
Add discovery config to qa-covid19

### PR DESCRIPTION
**Pull request by nss10**
Link to Jira ticket if there is one: [COV-1104](https://occ-data.atlassian.net/browse/COV-1104)

### Environments
- QA-Covid19

### Description of changes
- Add Discovery config to QA-Covid19 env

### CODE REVIEW

File: qa-covid19.planx-pla.net/portal/gitops.json [hunk 1]
```diff
@@ -1266,6 +1273,164 @@
...
+            "includeName": false,
+            "includeIfNotAvailable": false
+          }]
+        }
+      ]
+    },
+    "minimalFieldMapping": {
+      "tagsListFieldName": "tags",
+      "authzField": "auth_resource_path",
+      "uid": "_publication_id"
```
Comment (paulineribeyre) [inline qa-covid19.planx-pla.net/portal/gitops.json:1425]: what is `_publication_id`? is that what you chose as a unique ID to submit to the MDS?

File: qa-covid19.planx-pla.net/manifest.json [hunk 2]
```diff
@@ -4,33 +4,33 @@
     "That's all I have to say"
   ],
   "versions": {
+    "ambassador": "quay.io/datawire/ambassador:1.4.2",
```
Comment (paulineribeyre) [inline qa-covid19.planx-pla.net/manifest.json:7]: is there a change to the versions, or only the order? i don't see the change if there's one

File: qa-covid19.planx-pla.net/portal/gitops.json [hunk 3]
```diff
@@ -1266,6 +1273,164 @@
...
+      "header": {
+        "field": "title"
+      },
+      "fieldsToShow": [{
+          "groupName": "Publication Identifiers",
+          "includeName": false,
+          "fields": [
+          {
+            "name": "Authors",
+            "field": "authors",
```
Comment (paulineribeyre) [inline qa-covid19.planx-pla.net/portal/gitops.json:1356]: does the discovery page not have logic to capitalize names automatically?

File: qa-covid19.planx-pla.net/manifest.json (hunk 2 above)
Comment (nss10) [reply qa-covid19.planx-pla.net/manifest.json:7]: Just the order. I've sorted them in alphabetical order.

File: qa-covid19.planx-pla.net/portal/gitops.json (hunk 1 above)
Comment (nss10) [reply qa-covid19.planx-pla.net/portal/gitops.json:1425]: We can change it later if PRC has different requirements.

File: qa-covid19.planx-pla.net/portal/gitops.json (hunk 3 above)
Comment (nss10) [reply qa-covid19.planx-pla.net/portal/gitops.json:1356]: Let me check and get back to you.

Comment (nss10) [reply qa-covid19.planx-pla.net/portal/gitops.json:1356]: Oh! I missed to get back. Portal does do the auto capitalization.

Comment (paulineribeyre) [reply qa-covid19.planx-pla.net/portal/gitops.json:1356]: great, let's remove the `name` fields that are only doing that then

Comment (nss10) [reply qa-covid19.planx-pla.net/portal/gitops.json:1356]: Not really, I tried taking off the name fields, but that totally removed the column title in the `table view`

Comment (nss10) [reply qa-covid19.planx-pla.net/portal/gitops.json:1356]: The column fields that appear as column headers are all in ALL_CAPS. That's what I thought you meant.

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**